ob_intern/ViewController.swift

```diff
@@ -41,12 +41,14 @@ class ViewController: UIViewController {
     @IBAction func buttonToPost(_ sender: Any) {
         let controller = ShortVideoPostViewController.newInstance()
         self.navigationController?.pushViewController(controller, animated: false)
```

Comment:
พวกเปิด ปิดหน้าซอฟต์ใส่ animated ได้นะมันจะลื่นไหลขึ้น
```suggestion
        self.navigationController?.pushViewController(controller, animated: true)
```